src/spec/capitalize_spec.rb

```diff
@@ -0,0 +1,10 @@
+require_relative '../capitalize_decorator'
+require_relative '../person'
+
+describe CapitalizeDecorator do
+  it 'it should return name in caplitalize form' do
+    person = Person.new(30, 'shahadat hossain')
+    capitalize_person = CapitalizeDecorator.new(person)
+    expect(capitalize_person.correct_name).to eql('Shahadat hossain')
+  end
+end
```

Comment:
- [ ] As required for the project, Kindly make sure you add tests to all classes that you created while implementing the **Decorator** design pattern. Right now, I noticed that some classes miss tests with their `methods` respectively, I suggest you add them(i.e. tests for `Decorator`, `Nameable`, classes and `their methods respectively` )